src/main/java/com/xceptance/neodymium/util/LighthouseUtils.java

```diff
@@ -0,0 +1,217 @@
+package com.xceptance.neodymium.util;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.Assert;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WindowType;
+
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.jayway.jsonpath.JsonPath;
+import com.jayway.jsonpath.PathNotFoundException;
+
+import io.qameta.allure.Allure;
+
+/**
+ * Powered by <a href="https://developer.chrome.com/docs/lighthouse/overview?hl=de">Lighthouse</a> (Copyright Google)
+ */
+public class LighthouseUtils
+{
+    /**
+     * Creates a <a href="https://developer.chrome.com/docs/lighthouse/overview?hl=de">Lighthouse</a> report
+     * (Copyright Google) of the current URL and adds it to the Allure report.
+     * 
+     * @param driver
+     *            The current webdriver
+     * @param reportName
+     *            The name of the Lighthouse report attachment in the Allure report
+     * @throws Exception
+     */
+    public static void createLightHouseReport(WebDriver driver, String reportName) throws Exception 
```

Comment:
We should make the interface as simple as possible. There is no need to have the webdriver given as a parameter. Just use `Neodymium.getWebdriver()` inside this method.